projects/sbb-esta/angular-business/src/lib/contextmenu/contextmenu/contextmenu.component.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  AfterContentInit,
+  Component,
+  ContentChild,
+  OnInit,
+  ViewChild,
+  ViewEncapsulation
+} from '@angular/core';
+import { DropdownComponent, DropdownTriggerDirective } from '@sbb-esta/angular-public';
+
+@Component({
+  selector: 'sbb-contextmenu',
+  templateUrl: './contextmenu.component.html',
+  styleUrls: ['./contextmenu.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class ContextmenuComponent implements OnInit, AfterContentInit {
+  /**
+   * Reference to a dropdown instance.
+   */
+  @ContentChild(DropdownComponent, { static: true }) dropdown: DropdownComponent;
```

Comment:
Rename to _dropdown and change the comment to  /** @docs-private */, since this is an internal detail